scripts/tests/test_tiling.py

```diff
@@ -0,0 +1,714 @@
+''' Unit tests for tiling module. '''
+
+import os
```

Comment:
It doesn't appear these tests got run in CircleCI. Do you need to add the folder to the config?

![image](https://user-images.githubusercontent.com/32075205/115913477-82cab500-a425-11eb-9346-03813ea11c15.png)

Reply:
Done